src/ui/Typography.tsx

```diff
@@ -24,9 +24,14 @@ export const H4 = styled.h4`
 
 export const Subheading1 = styled.h5`
   font-size: var(--font-size-medium);
+  font-family: var(--font-heading-family);
```

Comment:
From what I can see in Figma spec This is Gesta Bold 20px (labeled as pt but should be px)

Netlify demo site might styled it differently but the Figma one looks more like the intended design

Reply:
Subheading2 is, but Subheading1 is not though if you check carefully it's definitely a monospace, both in the Netlify and the Figma in fact.

Reply:
I have updated these, hopefully the subheadings situation make more sense now 🙂